src/components/Notification/Notification.mdx

```diff
@@ -95,10 +98,28 @@ recognized and announced by assistive technology.
 
 ## Notification Heading
 
-The `Notification` component can be rendered with and without a heading.
+The `Notification` component can be rendered with and without a heading. By
+default, when a title is passed to the `notificationHeading` prop, the
+`Notification` component will render an `h4` element. If this causes accessibility
```

Comment:
It might be a good idea to also mention that it will be an `h4` with the `size` set to `heading6`.  In fact, no mater what level is passed, the `size` will be set to `heading6`, so the docs for the custom headings should also mention this.

Reply:
Yes, this is set in the component but not in the docs, so will update the docs.